DEVELOP.md

```diff
@@ -9,65 +9,66 @@ Dependencies:
 4. [RabbitMQ](http://www.rabbitmq.com/) for workers
 
 
-### Setting up CycleDash
+### 1. Install Dependencies
+
+```bash
+brew install Python                #brew will install the latest Python 2.7.9, which includes pip
+brew install postgres				
+brew install rabbitmq				
+pip install virtualenv
+```
+
+
+### 2. Set up CycleDash
 
 ```bash
 virtualenv venv                    # Initialize a new virtual environment.
-source venv/bin/activate           # Activate your virtual environment.
+source venv/bin/activate           # Activate your virtual environment. This is the only command you will need to run every time you start the app.
```

Comment:
Instead of noting this in a comment, how about adding a second paragraph like:

After the initial setup, the only command you'll need to run is:

```bash
source venv/bin/activate
```